gtfs/spec/en/reference.md

```diff
@@ -275,6 +275,8 @@ The [calendar_dates.txt](#calendar_datestxt) table explicitly activates or disab
 
 File: **Optional**
 
+There are two modelling options for describing fares. GTFS-Fares V1 is the legacy option for describing minimal fare information. GTFS-Fares V2 is an updated method that allows for a more detailed account of an agency's fare structure. Both are allowed to be present in a dataset, but only one method should be used by a data consumer for a given dataset. It is recommended that GTFS-Fares V2 takes precedence over GTFS-Fares V1. <br><br>GTFS-Fares V1: <br>-`fare_attributes.txt`<br>-`fare_rules.txt`<br><br>GTFS-Fares V2: <br>-`fare_leg_rules.txt`<br>-`fare_transfer_rules.txt`
```

Comment:
I would change it to: "For consumers it is recommended that GTFS-Fares V2 takes precedence over GTFS-Fares V1", and I would also add either: "For producers it is recommended to provide GTFS-Fares-V1 files for backwards compatibility" or "For producers that have/had GTFS-Fares-V1 files it is recommended not to remove them for backwards compatibility" (of course it also means they should also maintain them to keep the V1 up-to-date as much as possible within the possibilities of V1)

Reply:
Does this instruct producers who have used V1 to now provide both? That seems demanding on the producer side. I would favour keeping the recommendation general and leaving it up to producers and consumers to talk to eachother about which version they support. This would leave more room for adoption and evolution IMHO.

Reply:
See in one of my previous comments from a few minutes ago why the "producers talk to consumers" is not realistic when we talk about publicly available GTFS feeds.